internal/indexmanagement/reconcile.go

```diff
@@ -336,13 +336,27 @@ func (imr *IndexManagementRequest) reconcileIndexManagementCronjob(policy apis.I
 		{Name: "POLICY_MAPPING", Value: mapping.Name},
 	}
 
-	if policy.Phases.Delete != nil {
+	if policy.Phases.Delete != nil { //#BEE: I think I need to read namespace value here
+		var namespaceNames []string
+		var minAgeValue string
+		for _, namespaces := range policy.Phases.Delete.Namespace { //#BEE: I dont know why policy.Phases.Delete.Namespace is an array
+			namespaceNames = namespaces.NamespacesToDelete //#BEE: this is already an array..
+			minAgeValue = string(namespaces.MinAge)
+		}
+
+		namespaceNamesJson, err := json.Marshal(namespaceNames)
+		if err != nil {
+			return err
+		}
+
 		minAgeMillis, err := calculateMillisForTimeUnit(policy.Phases.Delete.MinAge)
 		if err != nil {
 			return err
 		}
 		envvars = append(envvars,
 			corev1.EnvVar{Name: "MIN_AGE", Value: strconv.FormatUint(minAgeMillis, 10)},
+			corev1.EnvVar{Name: "NAMESPACE_NAME", Value: string(namespaceNamesJson)}, //#BEE: now I can use this in the delete script
+			corev1.EnvVar{Name: "MIN_AGE_DBQ", Value: minAgeValue},
```

Comment:
Assuming that we have a list of namespaces specs, a single `MIN_AGE_DBQ` is not correct here. I suggest you pass create a small dict structure called `namespaceSpecsJson` that gives a good handle to execute a Delete-By-Query per Namespace, e.g.:
```json
{
  "namespace_a": "1d",
  "namespace_b": "2d",
},